pkg/system/filesys_windows.go

```diff
@@ -0,0 +1,64 @@
+// +build windows
+
+package system
+
+import (
+	"os"
+	"regexp"
+	"syscall"
+)
+
+// MkdirAll implementation that is volume path aware for Windows.
+func MkdirAll(path string, perm os.FileMode) error {
+	if re := regexp.MustCompile(`^\\\\\?\\Volume{[a-z0-9-]+}$`); re.MatchString(path) {
+		return nil
+	}
+
+	// The rest of this method is copied from os.MkdirAll and should be kept
+	// as-is to ensure compatibility.
```

Comment:
os.MkdirAll has a bug in that it doesn't correctly handle paths on Windows when they use volume paths as the root instead of drive letters.  For example, instead of "C:\foo" the path would be "\\?\Volume{3eb4b542-ef91-11e4-aa47-10604b6d8e85}\Foo".  Without the added regexp, the function thinks that the first folder that needs to be created is "\?" and fails.